pandas/core/indexes/base.py

```diff
@@ -3949,6 +3949,10 @@ def _evaluate_compare(self, other):
                         result = _comp_method_OBJECT_ARRAY(
                             op, self.values, other)
                 else:
+                    if isinstance(other, ABCSeries):
```

Comment:
What are the potential types of `other` here? `Series`, `Index`, anything else? If it's just those two, I think unconditionally doing `other = other._values` will work. If `other` may also be an `ndarray` then ignore this.

Reply:
This was an unsuccessful attempt to debug a failing appveyor build.  Let's circle back to the `other._values` idea once that problem is sorted out.